components/mixins/profile/profile-telecom-subscription-details.schema.json

```diff
@@ -0,0 +1,472 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/profile/profile-telecom-subscription-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Telecom Subscription",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": ["telecom"]
+  },
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "description": "Collects all generic details related to a customer's telecom subscription plan, including pricing, packages, and individual product subscriptions.",
+  "definitions": {
+    "telecom-subscriptions": {
+      "type": "object",
+      "properties": {
+        "xdm:telecomSubscription": {
+          "type": "object",
+          "properties": {
+            "xdm:subscriptionID": {
+              "title": "Subscription ID",
+              "type": "string",
+              "description": "Identifier for subscription."
+            },
+            "xdm:subscriptionDetails": {
+              "title": "Subscription Details",
+              "$ref": "https://ns.adobe.com/xdm/context/subscription",
+              "description": "Extension of the subscription data type to include subscription length, fees, status, etc."
+            },
+            "xdm:activationDate": {
```

Comment:
We already have start and end dates in the "https://ns.adobe.com/xdm/context/subscription" datatype. Do we still need activation and deactivation dates here ? Also, if it makes sense, can we move the common properties from these arrays to the subscription data type ?